index.html

```diff
@@ -2101,7 +2101,7 @@ <h4>
       <p its-locale-filter-list="zh-hant" lang="zh-hant">中文排版中，漢字與拉丁字母、希臘字母或阿拉伯數字等西文混排的狀況經常出現，以下列舉部分範例。</p>
       <ol>
         <li id="id105">
-          <p its-locale-filter-list="en" lang="en">One letter of European alphabet used as a symbol for something, such as ‘A’ or ‘B’.</p>
+          <p its-locale-filter-list="en" lang="en">An European alpha used as a symbol for something, such as ‘A’ or ‘B’.</p>
```

Comment:
"An" should be "A" here. See https://www.pristineword.com/grammar-a-european/

Reply:
Thanks. Fixed by [063ddfb](https://github.com/w3c/clreq/pull/492/commits/063ddfbe28641e618c8540eeed3711e4a670ea40).